integration-libs/s4-service/order/components/reschedule-service-order/reschedule-service-order.component.html

```diff
@@ -0,0 +1,98 @@
+<form [formGroup]="form" (submit)="rescheduleServiceOrder()">
+  <div class="cx-reschedule-service-form-fields">
+    <div class="cx-section-header d-none d-lg-block d-xl-block">
+      <h2>{{ 'rescheduleService.headerLabel' | cxTranslate }}</h2>
+    </div>
+
+    <div class="cx-section-content">
+      <p>{{ 'rescheduleService.contentLabel' | cxTranslate }}</p>
+      <div class="row">
+        <label
+          class="col-md-6"
+          [attr.aria-label]="'rescheduleService.datePickerLabel' | cxTranslate"
+        >
+          <span class="label-content">
+            {{ 'rescheduleService.datePickerLabel' | cxTranslate }}
+          </span>
+          <cx-date-picker
+            [control]="$any(form.get('scheduleDate'))"
+            [min]="(minServiceDate$ | async) ?? undefined"
+            label="rescheduleService.datePickerLabel"
+          >
+          </cx-date-picker>
```

Comment:
```suggestion
          <cx-date-picker
            [control]="$any(form.get('scheduleDate'))"
            [min]="(minServiceDate$ | async) ?? undefined"
            label="rescheduleService.datePickerLabel"
          />
```
`major:` Let's use self-closing tags for void components. See: https://wiki.one.int.sap/wiki/display/spar/Coding+Guidelines#CodingGuidelines-Useself-closingcomponenttags